config.toml

```diff
@@ -318,6 +318,7 @@ Patterns = [
 	{coords = [1479, 1758, 12], vehicle = 519, allowed = ["planes"], denied = ["rc"]},
 	{coords = [2322, 73, 26], vehicle = 523, move = [0, 18, 0, 0], allowed = ["cars", "bikes", "helis"]}, # Catalina (STB)
 	{coords = [2323, 70, 26], vehicle = 523, move = [0, 18, 0, 0], allowed = ["cars", "bikes", "helis"]}, # Player (STB)
+	{vehicle = 468, allowed = ["bikes"]},

```

Comment:
Hello, thanks for the pull request - this change affects a large number of missions other than Kickstart - including Wrong Side of the Tracks, Snail Trail, Dirt Track, Badlands among others.

I suggest adding either a threads check or a coordinate check for the bike to make it specific to Kickstart.

Also - config.toml is the file that's packaged as the default config in Rainbomizer release builds. You also have to change `config_default.hh` in `include/` to change the default config that's generated by the ASI itself.

Reply:
What is the conventional way of determining coordinates?
Do I have to reverse engineer the game or there is a tool?
I'm more than happy to do so but I don't know how to find those coordinates